intelmq/bots/BOTS

```diff
@@ -653,6 +103,45 @@
                 "xmpp_server": "xmpp server",
                 "xmpp_user": "xmpp username"
             }
+        },
+        "AlienVault OTX": {
```

Comment:
Any ideas on how to drop the sorting from the test while still checking for indentation and other style specifications?